helm.sh

```diff
@@ -0,0 +1,64 @@
+#!/bin/bash
+# This file is part of MinIO DirectPV
+# Copyright (c) 2023 MinIO, Inc.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+declare BUILD_VERSION KUBECTL_DIRECTPV
+
+function init() {
+    if [ "$#" -ne 1 ]; then
+        cat <<EOF
+USAGE:
+  helm.sh <VERSION>
+
+EXAMPLE:
+  $ helm.sh 4.0.6
```

Comment:
```suggestion
  $ helm.sh 4.0.7
```

as v4.0.7 is the latest version as of now